clients/unshaded/src/main/java/tachyon/client/RawTableMasterClient.java

```diff
@@ -61,6 +59,17 @@ protected String getServiceName() {
   @Override
   protected void afterConnect() {
     mClient = new RawTableMasterService.Client(mProtocol);
+    if (mServerVersion == -1) {
+      try {
+        mServerVersion = mClient.version();
+        if (!mCompatibleVersions.contains(mServerVersion)) {
+          throw new RuntimeException(ExceptionMessage.INCOMPATIBLE_VERSION.getMessage(
+              Constants.RAW_TABLE_MASTER_SERVICE_VERSION, mServerVersion));
+        }
+      } catch (TException e) {
```

Comment:
Oh I see, could we change the call to `getServerVersion`? It seems like getting the client version at the moment.